Cafeteria/Rating/__init__.py

```diff
@@ -20,11 +20,18 @@ def rateMenu(self, mealId):
 
         # if MongoDatabase.getMeal(mealId = mealId) == None:
         #     return "Error: Meal Doesn't Exist."
-
+        comment = request.values.get("comment") or None

```

Comment:
It guarantees variable comment gets None if the request.values.get('comment') returns [], "", 0 etc.